plone/recipe/varnish/recipe.py

```diff
@@ -27,7 +27,7 @@
 """
 
 COOKIE_PASS_DEFAULT = """\
-"auth_token|__ac(|_(name|password|persistent))=":"\.(js|css|kss)$"
+"auth_token|__ac(|_(name|password|persistent))=":"\.(js|css|kss|woff|woff2|eot|ttf|otf)$"
```

Comment:
Is kss still a thing ;) ? I propose to remove it.

Reply:
Just today I saw a stackoverflow comment that for the web woff and woff2 should be enough for reasonably current browsers. I think that was even a comment from 2019. So we could skip the others. But I dont mind.

Reply:
I was surprised that our new deployment of our main website had plenty of requests to fonts on `ttf`, `eot` and `svg` formats, given that `woff` and `woff2` were offered as well 🤯 

I did not dig which kind of browsers were though.